pkg/util/mount/mount.go

```diff
@@ -78,12 +78,15 @@ func (mounter *SafeFormatAndMount) formatAndMount(source string, target string,
 		// It is possible that this disk is not formatted. Double check using diskLooksUnformatted
 		notFormatted, err := mounter.diskLooksUnformatted(source)
 		if err == nil && notFormatted {
+			args := []string{source}
 			// Disk is unformatted so format it.
 			// Use 'ext4' as the default
 			if len(fstype) == 0 {
 				fstype = "ext4"
 			}
-			args := []string{"-E", "lazy_itable_init=0,lazy_journal_init=0", "-F", source}
+			if fstype == "ext4" || fstype == "ext3" {
+				args = []string{"-E", "lazy_itable_init=0,lazy_journal_init=0", "-F", source}
```

Comment:
> shouldn't this all go in a *_linux .go file?

Ah yes they should. I filed https://github.com/kubernetes/kubernetes/issues/16729. Would you be okay with me moving them after this PR and #14200 ?